python/snewpy/flavor.py

```diff
@@ -1,6 +1,7 @@
 import enum
 import numpy as np
 import typing
+import snewpy.utils
 
 class EnumMeta(enum.EnumMeta):
```

Comment:
Not a new change in this PR, but probably this child class should be called `FlavorEnumMeta` to avoid the possibility of an inadvertent name clash later:

```
from enum import EnumMeta       # did we mean to use this?
from flavor import EnumMeta     # or this?
```